### ISSUE TITLE
[Other Issues]: Nice to Haves

### ISSUE THREAD

**Issue by Wagnerp** [Krypton-Suite/Extended-Toolkit#28]
https://www.codeproject.com/Articles/13123/MessageBoxManager-A-Windows-Forms-component-that-p

https://www.codeproject.com/Articles/19865/How-to-extend-the-native-MessageBox-dialog-in-NET

https://www.codeproject.com/Articles/9656/Dissecting-the-MessageBox?q=messagebox+extended

https://www.codeproject.com/Articles/454307/A-versatile-MessageBox-replacement

https://www.codeproject.com/Articles/19566/Extend-OpenFileDialog-and-SaveFileDialog-the-easy

ToolTip / BalloonTip Links:
https://help.syncfusion.com/windowsforms/supertooltip/supertooltip

https://www.codeproject.com/Articles/98967/A-ToolTip-with-Title-Multiline-Contents-and-Image

https://www.grapecity.com/en/controls/winforms/supertooltip

https://www.codeproject.com/Articles/31411/VS-Style-Tooltips-and-A-Whole-Lot-More

http://www.c-sharpcorner.com/UploadFile/deepak.sharma00/how-to-show-notifyicon-in-windows-forms-application-using-C-Sharp/

http://www.hardcodet.net/wpf-notifyicon

Enhanced progress bar

https://www.codeproject.com/Articles/31406/Add-the-Percent-or-Any-Text-into-a-Standard-Progre

https://www.codeproject.com/Articles/48419/A-simple-custom-progressbar

https://www.codeproject.com/Articles/29457/TSmartProgressBar-A-Smart-C-ProgressBar-with-Perce

https://www.codeproject.com/Articles/1082902/How-to-Paint-on-Top-of-a-ProgressBar-using-Csharp

https://www.codeproject.com/Articles/33971/ProgressBar-with-Percentage

https://www.codeproject.com/Tips/292948/Glass-Style-ProgressBar

https://www.codeproject.com/Articles/834522/An-universal-Desktop-Splash-Dialog-with-Progressba

https://www.codeproject.com/Articles/32404/Custom-Control-for-Text-Over-a-Progress-Bar

http://www.dreamincode.net/forums/topic/94631-add-the-percent-into-a-progress-bar-updated/

https://github.com/Behzadkhosravifar/BlurMessageBox

In Chinese: https://github.com/ahdung/MessageBoxEx

https://blogs.msdn.microsoft.com/knom/2007/03/12/vista-command-link-control-with-c-windows-forms/

https://www.codeproject.com/Articles/1020671/MRU-File-Manager-Component

https://github.com/saleyn/ToolStripEditor

https://www.codeproject.com/Articles/29154/CodeTextBox-another-RichTextBox-control-with-synta

https://visualstudiomagazine.com/articles/2011/08/01/pfint_tear-off-tearstrips.aspx

https://www.codeproject.com/Articles/1029499/ToggleSwitch-Winforms-Control?

https://www.codeproject.com/Tips/794577/Windows-Forms-ToggleButton

**Comment by dax-leo** [comment]
Is it ok I drop few ideas here ? 😏

1. Radial Menu
http://www.devcomponents.com/kb2/?p=1363

2. Nuget packages are kind of messy. Any cleanup plans ?
![image](https://user-images.githubusercontent.com/70173560/96185124-b9d18280-0f39-11eb-99da-e4387959d0d9.png)

**Comment by dax-leo** [comment]
Title menu bar (KryptonForm?)

Just like they do it for DevExpress Forms:
https://docs.devexpress.com/WindowsForms/DevExpress.XtraBars.ToolbarForm.ToolbarForm
![image](https://user-images.githubusercontent.com/70173560/96185934-f94c9e80-0f3a-11eb-963d-1162eea96933.png)

https://stackoverflow.com/questions/2575216/how-to-move-and-resize-a-form-without-a-border
https://stackoverflow.com/questions/7482922/remove-the-title-bar-in-windows-forms
http://csharphelper.com/blog/2015/07/move-a-form-without-a-title-bar-in-c/

My initial attempt with sizable border. Win10 shows ugly line on top so I used suggestion from this link: https://stackoverflow.com/questions/2575216/how-to-move-and-resize-a-form-without-a-border
![image](https://user-images.githubusercontent.com/70173560/96188888-d375c880-0f3f-11eb-8079-b0131a8c6ac0.png)

Still ugly but I just wanted to check if it is manegable.

**Comment by Wagnerp** [comment]
Hi @dax-leo 

Re Radial Menus: Yes, I've found [this](https://github.com/Julien-Marcou/RadialMenu), but I don't know how to convert WPF to WinForms.

**Comment by Wagnerp** [comment]
> Is it ok I drop few ideas here ? 😏
> 
>    Nuget packages are kind of messy. Any cleanup plans ?
>    ![image](https://user-images.githubusercontent.com/70173560/96185124-b9d18280-0f39-11eb-99da-e4387959d0d9.png)

Yep, all legacy packages will be deprecated as of November. I don't know if I can upload the current NuGet packages to the GitHub feed.

**Comment by Wagnerp** [comment]
Circular progress bar: https://github.com/falahati/CircularProgressBar

**Comment by Wagnerp** [comment]
Exception messagebox: https://www.codeproject.com/Articles/18838/Exception-Message-Box-Using-C

![image](https://user-images.githubusercontent.com/949607/97077265-243d8f00-15da-11eb-8ac4-37464f611d42.png)

**Comment by Wagnerp** [comment]
Hi @dax-leo 

Do you mean something like this: https://www.codeproject.com/Articles/32623/Vista-Aero-ToolStrip-on-Non-Client-Area

**Comment by Wagnerp** [comment]
https://github.com/dotnet/winforms-datavisualization

**Comment by Smurf-IV** [comment]
Support for `FontAwesome ttf` so that icons can be used - everywhere an image can be used - i.e. in Navigator tabs

**Comment by Wagnerp** [comment]
This might be interesting https://stackoverflow.com/questions/56481230/how-to-create-windows-10-acrylic-transparency-effect-in-winform-c-sharp

**Comment by Wagnerp** [comment]
https://github.com/IgnaceMaes/MaterialSkin

**Comment by Wagnerp** [comment]
https://github.com/ArthurHub/HTML-Renderer

**Comment by Smurf-IV** [comment]
> https://github.com/ArthurHub/HTML-Renderer

Isn't this already covered by the following:
![image](https://user-images.githubusercontent.com/2418812/218255251-638d819f-112a-4def-ad41-a6325a6f687f.png)

**Comment by Wagnerp** [comment]
> > https://github.com/ArthurHub/HTML-Renderer
> 
> Isn't this already covered by the following: ![image](https://user-images.githubusercontent.com/2418812/218255251-638d819f-112a-4def-ad41-a6325a6f687f.png)

Not the ability to have hyperlinks in tooltips though?

**Comment by Smurf-IV** [comment]
> Title menu bar (KryptonForm?)
> 
> Just like they do it for DevExpress Forms: https://docs.devexpress.com/WindowsForms/DevExpress.XtraBars.ToolbarForm.ToolbarForm ![image](https://user-images.githubusercontent.com/70173560/96185934-f94c9e80-0f3a-11eb-963d-1162eea96933.png)
> 
> https://stackoverflow.com/questions/2575216/how-to-move-and-resize-a-form-without-a-border https://stackoverflow.com/questions/7482922/remove-the-title-bar-in-windows-forms http://csharphelper.com/blog/2015/07/move-a-form-without-a-title-bar-in-c/


Manage most of the concerns from this as well:
https://github.com/rossy/borderless-window

**Comment by Wagnerp** [comment]
https://github.com/Taiizor/ReaLTaiizor

**Comment by Wagnerp** [comment]
> Support for `FontAwesome ttf` so that icons can be used - everywhere an image can be used - i.e. in Navigator tabs

@Smurf-IV Would this be a good candidate for a ST feature request (future version)?
